website/project/model.py

```diff
@@ -3308,6 +3297,16 @@ def get_admin_contributors_recursive(self, unique_users=False, *args, **kwargs):
                     else:
                         yield (contrib, node)
 
+    def get_admin_contributors(self, users):
+        """Return a set of all admin contributors for this node and admin
+        contributors on descendant nodes. Excludes contributors on node links and
```

Comment:
This docstring does not match the actual behavior of this method; admin contributors on descendant nodes will not be returned. Given that we already have `get_admin_contributors_recursive`, I assume this method isn't supposed to return admins on descendants. If that's the case, please update the docstring.